go-controller/pkg/ovn/ovn.go

```diff
@@ -104,7 +119,13 @@ func (oc *Controller) WatchEndpoints() {
 				logrus.Errorf("Error in adding load balancer: %v", err)
 			}
 		},
-		UpdateFunc: func(old, new interface{}) { return },
+		UpdateFunc: func(old, new interface{}) {
+			ep := new.(*kapi.Endpoints)
+			err := oc.addEndpoints(ep)
```

Comment:
This would need a deep equal too I guess. On second thoughts I am warming to having a cache maintained locally for sync or debug purposes.